tests/romtools/test_snapshots_mpi.py

```diff
@@ -44,6 +44,9 @@ def test_mpi_snapshots():
         matrix = sd.getSnapshotsAsArray()
         assert matrix.shape[0] == myGids.shape[0] * sd.getNumVars()
         assert matrix.shape[1] == 5
+    else:
+        if rank == 0:
+            print(f"Test skipped with np = {comm.Get_size()} (requires np = 3)")
```

Comment:
can you please make a helper function for doing this that lives inside the tests in a `helpers.py` or similar 
```py
void mpi_skipped_test_mismatching_commsize(mpiComm, testName: str, npNeeded: int)
{
  if mpiComm.Get_rank() == 0:
     print(f"Test {testname} skipped for np = {mpiComm.Get_size()} (requires np = {npNeeded})")
}
```

Reply:
Done